numpy/linalg/tests/test_regression.py

```diff
@@ -90,6 +90,12 @@ def test_svd_no_uv(self):
                 assert_equal(np.linalg.matrix_rank(a), 1)
                 assert_array_less(1, np.linalg.norm(a, ord=2))
 
+    def test_norm_object_array(self):
```

Comment:
It would be good to test more of the norms, or at least all that work with this fix.